src/main/java/com/dev/cinema/model/dto/shoppingcart/ShoppingCartResponseDto.java

```diff
@@ -7,6 +7,5 @@
 @Data
 public class ShoppingCartResponseDto {
     private Long id;
-    private Long userId;
     private List<TicketResponseDto> tickets;
```

Comment:
same for ShoppingCart - let's return ticketIds and remove ticketDto and mapper

Reply:
done